content/javascript/concepts/data-types/data-types.md

```diff
@@ -29,4 +29,55 @@ Data types are the classifications we give to the different kinds of data that w
 
 The first 7 of those types are considered primitive data types. They are the most basic data types in the language. Objects are a little more complex.
 
-At first, eight types may not seem like that many, but soon you’ll observe the world opens with possibilities once you start leveraging each one.
+At first, eight types may not seem like that many, but they are the foundation that helps programmers to build a whole new world.
+
+### Examples
+
+* Number and BigInt
+
+```js
+var num     = 7;
+var bigNum  = 9999999999999999n;
+```
+
+BigInt is necessary for big numbers because they are unreliable with Number type:
```

Comment:
This might be better as a comment in the below code block. All the other descriptive text is in code comments.